--- QupZilla-2.2.1/src/lib/webtab/searchtoolbar.ui.omv~	2017-11-03 18:21:43.234411318 +0100
+++ QupZilla-2.2.1/src/lib/webtab/searchtoolbar.ui	2017-11-03 18:32:07.109065445 +0100
@@ -65,7 +65,7 @@
       </sizepolicy>
      </property>
      <property name="shortcut">
-      <string notr="true">Ctrl+Shift+G</string>
+      <string>Ctrl+Shift+G</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
@@ -81,7 +81,7 @@
       </sizepolicy>
      </property>
      <property name="shortcut">
-      <string notr="true">Ctrl+G</string>
+      <string>Ctrl+G</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
